### PR TITLE
Upgrade Evernote SDK

### PR DESCRIPTION
**Pull request by cmckni3**

### CODE REVIEW

**Comment by cmckni3** [comment]
@volkhin ☝️

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,8 @@
+root=true
```
Comment (volkhin) [inline .editorconfig]: thanks for the PR! Can you please remove this file?

**Review by volkhin — CHANGES_REQUESTED** [review]

File: .editorconfig (hunk 1 above)
Comment (cmckni3) [inline .editorconfig]: Sure. Do you not use [EditorConfig](http://editorconfig.org)?

Comment (volkhin) [inline .editorconfig]: no, actually I never tried it. Will take a look, thanks:)

**Pull request merged**